test/replication-luatest/recovery_triggers_test.lua

```diff
@@ -1,5 +1,5 @@
 local t = require('luatest')
-local cluster = require('test.luatest_helpers.cluster')
+local cluster = require('luatest.replica_set')
```

Comment:
btw, why is it 
```lua
local cluster = require('luatest.replica_set')
```
but not
```lua
local cluster = t.replica_set
```
?

Reply:
I think it's better to call it through require.

Reply:
cluster remain as a part of luatest and not just another local variable.

Reply:
Another question. Shouldn't the variable be renamed now that the class is renamed:
```suggestion
local replica_set = require('luatest.replica_set')
```

Reply:
I planned this as a separate PR, but if you insist, I can make one more commit.